tasks/lets_encrypt.yml

```diff
@@ -63,7 +63,7 @@
     challenge: "{{ lets_encrypt_challenge_type }}"
     agreement: "{{ lets_encrypt_agreement }}"
     csr: "/etc/ssl/requests/{{ lets_encrypt_resource_name }}.csr"
-    dest: "/etc/ssl/certs/{{ lets_encrypt_resource_name }}.crt"
+    dest: "{{ build_dir.path }}/01-{{ lets_encrypt_resource_name }}.crt"
```

Comment:
please use the convention that variable names should start with the role name

Reply:
Changed the variable name to follow the convention.